speech/google/cloud/speech/transcript.py

```diff
@@ -38,7 +38,7 @@ def from_api_repr(cls, transcript):
         :rtype: :class:`Transcript`
         :returns: Instance of ``Transcript``.
         """
-        return cls(transcript['transcript'], transcript['confidence'])
+        return cls(transcript.get('transcript'), transcript.get('confidence'))
```

Comment:
Actually, I don't think so. There would be no result at all. Maybe I should switch `transcript`  back to an index then?